### ISSUE TITLE
Declarations with deviceCertificate via BIG-IQ fail

### ISSUE THREAD

**Issue by HidingUnderMyRock** [F5Networks/f5-declarative-onboarding#297]
### Environment
 * Declarative Onboarding Version: 1.21.1-2 on BIG-IQ (1.28.0 on BIG-IP) - can test with 1.28.0 on IQ again if needed
 * BIG-IP Version: v15.1.x and v16.1.x
 * BIG-IQ 8.1.0.2

### Summary
DO declarations that assign deviceCertificate complete when sent direct to the BIG-IP, but timeout when sent via BIG-IQ.  Removing the deviceCertificate class section allows the declaration to succeed via BIG-IQ.  Sending a declaration that only includes the deviceCertificate class section results in a timeout via BIG-IQ, but succeeds when sent direct to the BIG-IP.

### Steps To Reproduce
1. Submit any declaration that attempts to set the device certificate via BIG-IQ:
```json
"deviceCertificate": {
    "class": "DeviceCertificate",
    "certificate": {
        "base64": "**snip**"
    },
    "privateKey": {
        "base64": "**snip**"
    }
}
```

2. Observe the following error response:
```json
Eventually the onboarding task will fail with BIG-IQ stating:

Task Failed: Failed to complete onboarding task. code: 401, message: invalid config - rolled back, error: tryUntil: max tries reached: tryUntil: max tries reached: <html> <head><title>401 Authorization Required</title></head> <body> <center><h1>401 Authorization Required</h1></center> <hr><center>webd</center> </body> </html>
```

### Expected Behavior
The declaration should complete with a deviceCertificate class via both BIG-IQ and direct to the BIG-IP.

### Actual Behavior
DO declarations that attempt to set the deviceCertificate via BIG-IQ are unsuccessful, the certificate is never set on the device, and the declaration times out as if IQ has no idea how to handle this process and simply gives up.

**Comment by dstokesf5** [comment]
Do you encounter any issues when submitting the declaration directly to a BIG-IP without using BIG-IQ?

**Comment by HidingUnderMyRock** [comment]
No, it succeeds if I send the declaration directly to the BIG-IP. It is only when sending the declaration via IQ that it fails. Removing *only* the deviceCertificate section allows the declaration to succeed. In fact, a simple declaration that only contains the deviceCertificate (via IQ) will fail.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-3057.

**Comment by sunitharonan** [comment]
This has been resolved in the 1.33.0 release of DO.
